utils/vlog/tests/test.mk

```diff
@@ -0,0 +1,32 @@
+#	xsltproc --xinclude ../../../../common/xml/xmlsort.xsl $< > $@.tmp
```

Comment:
This file should be removed.

Reply:
We have to remove the file from the compilation process or just the comment?

Reply:
`test.mk` should be removed as we use cmake now.